packages/core/src/stylable-processor.ts

```diff
@@ -975,7 +864,7 @@ export class StylableProcessor {
         value: any
     ) {
         const name = selector.replace('.', '');
-        const typedRule = this.meta.mappedSymbols[name] as ClassSymbol | ElementSymbol;
+        const typedRule = STSymbol.get(this.meta, name) as ClassSymbol | ElementSymbol;
```

Comment:
this.meta.getSymbol

Reply:
same as with the following comment on the `meta.getClass or meta.getTypeElement` to replace direct feature access. In the core I went directly to features without passing through the public APIs